frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -236,14 +237,6 @@ export default ({
     reply () {
       this.$emit('reply')
     },
-    copyMessageLink () {
```

Comment:
This existing 'copy' action can be handled inside `MessageActions.vue` directly (it's not necessary to keep this logic inside the parent component and handle it there, which requires emitting a custom-event from the child). So I thought I would DRY this logic while at this task.